pkg/store/copr/coprocessor.go

```diff
@@ -102,6 +102,7 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, variables any, op
 		return errRes
 	}
 	ctx = context.WithValue(ctx, tikv.RPCCancellerCtxKey{}, it.rpcCancel)
+	ctx = context.WithValue(ctx, util.RUDetailsCtxKey, util.NewRUDetails())
```

Comment:
why new `RUDetailsCtxKey` here, will it collect the stat with old path?, what is different with `clientutil.RUDetailsCtxKey`.

Reply:
> will it collect the stat with old path

In some places the incoming ctx does not have ru(Analyze), in others it does(executeKillStmt). I added the judgment

> no different with `different with clientutil.RUDetailsCtxKey.`

just the go import name is different